src/components/tl/dpu/tl_dpu.h

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2021.  ALL RIGHTS RESERVED.
+ *
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCC_TL_DPU_H_
+#define UCC_TL_DPU_H_
+
+// #include "tl_dpu_coll.h"
+#include "components/tl/ucc_tl.h"
+#include "components/tl/ucc_tl_log.h"
+#include <ucp/api/ucp.h>
+
+#ifndef UCC_TL_DPU_DEFAULT_SCORE
+#define UCC_TL_DPU_DEFAULT_SCORE 20
```

Comment:
Set different default SCORE. Defaults SCORES of different TL components must be different (otherwise there appears ambiguity in the TL selection - which TL to choose when scores are equal)